ros1_bridge_ws/docker/start-bridge.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+if [ $# = 1 ]; then
+    export ROS_MASTER_URI=$1
+else
+    export ROS_MASTER_URI=http://localhost:11311
+fi
+
+source /ros2_humble/install/setup.bash
+source /ros2_humble/install/ros1_bridge/share/ros1_bridge/local_setup.bash
```

Comment:
This may be a stupid question. Why can't we simply run:

```sh
source /ros2_humble/install/setup.bash
```

is it due to the build command with package select?

https://github.com/j3soon/ros2-agv-essentials/blob/7baa567c564e0c2a6e92c350c1973c3440cf1b0a/ros1_bridge_ws/docker/Dockerfile#L103-L104

Reply:
No. Normally, even if we compile the package separately, we should only need to source the `setup.bash` file. However, I don't understand why ROS 2 can't find ros1_bridge if I didn't source it manually.

The situation is similar to the one mentioned earlier. I stumbled upon this solution by accident."

Reply:
I suggest briefly mention in the comment that this command is necessary for ROS 2 to find ros1_bridge.